File: python/7 kyu/72.py
# https://www.codewars.com/kata/53921994d8f00b93df000bea/train/python

# Passed

def content_weight(bottle_weight, scale): 
    scaler = int(scale.split()[0])
    if scale.endswith("larger"):
        multiplier = scaler / (scaler + 1)
    else:
        multiplier = 1/ (scaler + 1)

    contents_weight = bottle_weight * multiplier
    return contents_weight

output = content_weight(1000, "49 times larger")
print(output)

File: nodejs/7 kyu/72.js
// https://www.codewars.com/kata/58f5c63f1e26ecda7e000029/train/javascript

// Passed

function wave(str) {
  const mexicanWave = str
    .split("")
    .map((char, idx) => {
      if (char === " ") return null;
      let mexicanWord =
        str.slice(0, idx) + str[idx].toUpperCase() + str.slice(idx + 1);
      return mexicanWord;
    })
    .filter((word) => word !== null);

  return mexicanWave;
}

const output = wave("hello");
console.log(output);